src/Options/Generic.hs

```diff
@@ -218,31 +234,52 @@ auto = do
     language extension
 -}
 class ParseField a where
-    parseField
-        :: Maybe Text
+        --(forall f b. Options.Mod f b)
+    parseFieldH
```

Comment:
Could you rename `parseFieldH` to `parseHelpfulField`?  The rationale is that I try to avoid single-letter abbreviations in function names